lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,17 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*:after,
+*:before {
+    box-sizing: inherit;
+}
+
+.main {
```

Comment:
Adding padding to this element creates a border-like effect.  You might consider adding the padding to the children elements inside .main instead of the parent.